creds.txt

```diff
@@ -0,0 +1,2 @@
+id=c4804250b5b441d08d5c4ec66a9e46f9
+secret=3c6a46407d8d47e79de88d8fa874fc2f
```

Comment:
Not a good idea to store API keys in a public place like this. You should have included this file in your `.gitignore`.